GECKO_REVISION

```diff
@@ -1 +1 @@
-423673:403d573285b9
+423684:831be24c0b0f
```

Comment:
We should try to update the Gecko revision in a stand alone commit, independent of other changes, so that it's clearer what's changed due to the Gecko revision update, and what's changed otherwise.